util/find_missing_translations.py

```diff
@@ -22,17 +22,22 @@
 non_zoneregex_locales = all_locales - zoneregex_locales
 
 # Where to start looking for files.
+
```

Comment:
Can you remove these additional newlines?

Reply:
sry this was auto-formatting of my sublime Q_Q